packages/auth/src/app.ts

```diff
@@ -353,7 +353,13 @@ export class App {
           signed: true,
           store: {
             async get(key) {
-              return await redis.hgetall(key)
+              const s = await redis.hgetall(key)
+              const session = {
+                nonce: s.nonce,
+                _expire: Number(s._expire),
+                _maxAge: Number(s._maxAge)
+              }
+              return session
```

Comment:
this would lose anything from `s` not explicitly set in `session`. not sure we can assume the only properties set on the session in redis will be these. we could do `session = { ...s, _maxAge: Number(s._maxAge), _expire: Number(s._expire)}` but that would have the same problem if the other fields are expected not to be strings (just like we saw with `maxAge`).

I think a more common way of getting/settings is `JSON.stringify(object)` on set and `JSON.parse(string)` on get, which which would take care of converting these to numbers without manually reforming the session object. hashes were suggested to me originally when I implemented this but not sure it's the best way to do it now...